app/src/main/java/com/infomaniak/mail/ui/main/folder/TwoPaneFragment.kt

```diff
@@ -50,28 +51,40 @@ abstract class TwoPaneFragment : Fragment() {
     fun isOnlyLeftShown() = slidingPaneLayout.let { it.isSlideable && !it.isOpen }
     fun isOnlyRightShown() = slidingPaneLayout.let { it.isSlideable && it.isOpen }
 
+    private val searchFolder by lazy {
+        Folder().apply {
+            id = FolderController.SEARCH_FOLDER_ID
+            name = getString(R.string.searchFolderName)
+        }
+    }
```

Comment:
Is it not strange to create a search Folder manually like this? Wouldn't a Pair or a data class be less confusing?

Reply:
We need a Folder object in several places for the rest of the logic.

Reply:
Fixed.